### ISSUE TITLE
Feat: modify path's name from popup without directly editing json

### ISSUE THREAD

**Issue by vague2k** [LintaoAmons/cd-project.nvim#11]
### allow modification of a path's name from popup without directly editing json
Inspired by #10

**Comment by LintaoAmons** [comment]
Yes, thanks for the idea, I think it's better integrated it with the telescope picker. add actions to it, like rename/delete an entry etc.

**Comment by vague2k** [comment]
This will probably get more attention after working on issue #10
